notebook/jstest.py

```diff
@@ -222,6 +222,7 @@ def __init__(self, section, xunit=True, engine='phantomjs', url=None):
         self.section = section
         self.xunit = xunit
         self.url = url
+        self.base_url = '/a@b/'
```

Comment:
What's `/a@b/` mean?  Sorry if I'm missing something obvious here.  Kind of looks like a regex!  Google foo failed me: https://www.google.com/#q=%2Fa%40b%2F